tests/manager/manager-ctor-001.phpt

```diff
@@ -1,12 +1,12 @@
 --TEST--
-MongoDB\Driver\Manager::__construct()
+MongoDB\Driver\Manager::__construct() with default URI
 --SKIPIF--
 <?php require __DIR__ . "/../utils/basic-skipif.inc"; CLEANUP(STANDALONE) ?>
 --FILE--
 <?php
 require_once __DIR__ . "/../utils/basic.inc";
 
-$manager = new MongoDB\Driver\Manager(STANDALONE);
+$manager = new MongoDB\Driver\Manager();
```

Comment:
`manager-ctor-001.phpt` was renamed to `manager-ctor-002.phpt`, so this is a new test.

The connection doesn't even happen until we do something that requires it, so this is basically just testing parsing of the default URI and the constructor allowing all params to be optional.